templates/default/hopssite/hs_rows.sql.erb

```diff
@@ -1,6 +1,6 @@
 insert into dataset_status select 0, "upload";
 insert into dataset_status select 1, "download";
-insert into variables select "dela_version", "0.1";
+insert into variables select "dela_version", "0.1.0";
```

Comment:
When and how is this script executed? Is it re-executed during an update? If so, you shouldn't use `insert into` as it will fail with a PK violation. Replace it with `REPLACE INTO`. See the rest of the dml files in /templates